### PR TITLE
Black Close button and related code 

### PR DESCRIPTION
**Pull request by Drudoo**
Added black close button and code that works with settings, tested and works

The image should be placed in layout>Library>Application Support>MobileNotifier 

Link to image http://cl.ly/2h3B1f0f3J3n0X423w42

When doing commits remotely the user apparently does not show up as my github user =/

Screenshot: http://cl.ly/3T140J1V0A1A3U350z46

### CODE REVIEW

**Comment by timnovinger** [comment]
@Drudoo, good idea for the new black version of the close image. Only one problem though....can you commit the new image itself (statusbar_alert_dismiss_black.png) into this pull request?

Then it will merge into the master branch cleanly. Thanks.

**Comment by Drudoo** [comment]
I tried many times but it just wont upload :/ I don't know how to do it online and remotely it just does not update.

**Comment by Drudoo** [comment]
I will try again and see what i can do =)

**Comment by timnovinger** [comment]
If you add it to `layout/Library/Application Support/MobileNotifier/` git should see the changes. You can then commit and push it.

**Comment by Drudoo** [comment]
Ok, one sec, let me try again =)

**Comment by Drudoo** [comment]
Worked =P Had to use \ to get the 'space'. A bit noobish to forget that =P

**Comment by timnovinger** [comment]
Awesome, good job dude!
